skymp5-server/cpp/lib_espm/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-file(GLOB ESPM_SRC "*.h" "*.cpp")
+file(GLOB ESPM_SRC "*.h" "*.cpp" "impl/*.cpp")
```

Comment:
I don't think we should introduce `impl` directory. This would be the first place in the project with such naming. And btw if we add this, we should consider moving more espm stuff to the `impl` folder to be consistent. `lib_espm` folder itself contains tons of implementation details. We just have no project-wide rules for handling this for now.

Reply:
I also feel that memory mapping might be used in other places in the future. So, maybe one day we leave `impl` empty.

Reply:
Seems reasonable, done that!